subsys/suit/provisioning/soc/Kconfig.nrf9280

```diff
@@ -0,0 +1,167 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+#
+# WARNING:
+#  All addresses and sizes in this file must be sunchronized with the SUIT storage address
+#  defined in the Device Tree.
+#  Any modification of those values require alignment in the SDFW.
+#  Contents of the MPI (vendor name, class name, policies) may be modified by the
+#  product vendor.
+#
+
+config SUIT_MPI_SOC_NRF9280
+	bool
+	default y if SOC_SERIES_NRF92X
+
+if SUIT_MPI_SOC_NRF9280
+
+manifest=ROOT
+manifest-class=nRF9280_sample_root
+manifest-downgrade-prevention-enabled=n
+manifest-independent-updates=y
+manifest-signature-check=DISABLED
+manifest-mpi-offset=0x2000
+
+menu "Root manifest configuration"
+
+# Define as a regular config, so it is impossible to disable generation of the
+# ROOT manifest configuration.
+config SUIT_MPI_$(manifest)
+	bool
+	default y
+
+rsource "Kconfig.template.manifest_config"
+
+endmenu
+
+manifest=APP_RECOVERY
+manifest-class=nRF9280_app_recovery
+manifest-downgrade-prevention-enabled=n
+manifest-independent-updates=y
+manifest-signature-check=DISABLED
+manifest-mpi-offset=0x2030
+
+menuconfig  SUIT_MPI_$(manifest)
```

Comment:
1 space after menuconfig/config, seems to be multiple times it has 2 in this file

Reply:
Fixed all of these.